lib/cisco_node_utils/ace_v4.rb

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # RouterAcl - node utility class for RouterAcl config mgmt.
+  class Ace < NodeUtil
+    attr_reader :acl_name, :seqno, :afi
+
+    # name: name of the router instance
+    # instantiate: true = create router instance
+    def initialize(acl_name, seqno, afi)
+      @acl_name = acl_name
+      @afi = afi
+      @seqno = seqno
+      @set_args = @get_args = { acl_name: @acl_name, seqno: @seqno }
+    end
+
+    # Create a hash of all current router instances.
+    def self.aces
+      instances = config_get('acl_v4', 'ace')
```

Comment:
If this provider is intended to support v6 as well then `self.aces` is not going to work very well.